Uno.Gallery/Uno.Gallery.Shared/Views/Styles/NeumorphicStyles.xaml

```diff
@@ -3196,7 +3196,8 @@
 		<Setter Property="ManipulationMode" Value="System,TranslateX" />
 		<Setter Property="FocusVisualMargin" Value="0" />
 		<Setter Property="UseSystemFocusVisuals" Value="False" />
-
+		<Setter Property="MinWidth" Value="0" />
```

Comment:
That's a weird value.

Reply:
That's related to this change made in Uno.Themes. https://github.com/unoplatform/Uno.Themes/pull/1237